mesonbuild/scripts/meson_install.py

```diff
@@ -283,7 +283,7 @@ def run_install_script(d):
             rc = subprocess.call(script + args, env=child_env)
             if rc != 0:
                 sys.exit(rc)
-        except:
+        except Exception:
```

Comment:
This probably should have been `subprocess.check_call()` and `subprocess.CalledProcessError`